src/compiler/tsbuild.ts

```diff
@@ -1788,31 +1788,25 @@ namespace ts {
 
         let reportQueue = true;
         let successfulProjects = 0;
-        let errorProjects = 0;
         while (true) {
             const invalidatedProject = getNextInvalidatedProject(state, buildOrder, reportQueue);
             if (!invalidatedProject) break;
             reportQueue = false;
             invalidatedProject.done(cancellationToken);
-            if (state.diagnostics.has(invalidatedProject.projectPath)) {
-                errorProjects++;
-            }
-            else {
-                successfulProjects++;
-            }
+            if (!state.diagnostics.has(invalidatedProject.projectPath)) successfulProjects++;

```

Comment:
Need test for this.. you can look at tests in `src/testRunner/unittests/tsbuild` for how to write them.

Reply:
Done.